inbox/mailsync/gc.py

```diff
@@ -59,15 +56,13 @@ def __init__(
         account_id,
         namespace_id,
         provider_name,
-        uid_accessor,
```

Comment:
This does not need to be passed into the initializer as we were passing it the same thing everywhere.